jcl/src/java.base/share/classes/java/lang/Access.java

```diff
@@ -518,7 +518,7 @@ public Thread currentCarrierThread() {
 		return Thread.currentCarrierThread();
 	}
 
-/*[IF JAVA_SPEC_VERSION >= 20]*/
+/*[IF JAVA_SPEC_VERSION >= 19]*/
```

Comment:
This is already inside `/*[IF JAVA_SPEC_VERSION >= 19]*/` from line 516. Which also makes the `[ELSE]` obsolete.